pytest_odoo.py

```diff
@@ -169,3 +169,7 @@ def _importtestmodule(self):
             )
         self.config.pluginmanager.consider_module(mod)
         return mod
+
+
+class OdooTestPackage(_pytest.python.Package, OdooTestModule):
+    pass
```

Comment:
please could you add some docstring to the class. Some context will be helpful.

Also, it would be better to have the declaration of the class before its usage. Easier for my lazy brain ;)

Reply:
done ;)